pages/[categoryId]/[subCategoryId]/[productSlug].tsx

```diff
@@ -40,11 +40,16 @@ export default function ProductPage({
         <div className="gel-layout__item gel-1/2@m">
           <h1 className="page-title product-page__title">{product.title}</h1>
           <p className="product-page__price">£{product.price}</p>
+          <Link href={product.url}>
+            <a className="product-page__buy gel-hide-gte@m">
+              Buy on Etsy
+            </a>
+          </Link>
           <p className="product-page__description">
             {htmlParser(product.description.replace(/\n/g, '<br />'))}
           </p>
           <Link href={product.url}>
-            <a className="product-page__buy">
+            <a className="product-page__buy gel-hide-lte@s">
               Buy on Etsy
             </a>
           </Link>
```

Comment:
Lets always keep the buy button at the top

Reply:
Done